Firmware/variants/1_75mm_MK3-EINSy10a-E3Dv6full.h

```diff
@@ -326,7 +331,11 @@
 #define  DEFAULT_Kp 21.70
 #define  DEFAULT_Ki 1.60
 #define  DEFAULT_Kd 73.76
-#else
+#elif HOTEND == 2 //E3D REVO PID values
```

Comment:
Please don't do this and add clutter to the files for defines that are never used in this context.

It just adds either more churn when every single file needs updating or it will get out of sync and cause even more headaches..

No #define hotend, just set the constants.